agent/config/config.go

```diff
@@ -455,6 +476,24 @@ func getContainerStartTimeout() time.Duration {
 	return containerStartTimeout
 }
 
+func getAgentPullBehavior() AgentPullBehaviorType {
```

Comment:
you should rebase on top my pr #1353 once it's merged so that this gets moved to `parse.go`.

Reply:
OK